internal/operands/data-sources/reconcile.go

```diff
@@ -396,7 +396,7 @@ func reconcileDataSource(dsInfo dataSourceInfo, request *common.Request) (common
 
 			foundDs := foundRes.(*cdiv1beta1.DataSource)
 			newDs := newRes.(*cdiv1beta1.DataSource)
-			if !dsInfo.autoUpdateEnabled || foundDs.Spec.Source.PVC == nil {
+			if !dsInfo.autoUpdateEnabled || (foundDs.Spec.Source.PVC == nil && foundDs.Spec.Source.Snapshot == nil) {
```

Comment:
I forget why SSP even needs to update the `PVC` here, shouldn't CDI keep the DS updated using the DIC?

Reply:
I guess the DataSources can be also managed by something else (dsInfo.autoUpdateEnabled=false),
in that case, if there's nothing there we put some defaults on?

Reply:
Yeah apologies now I remember, these are just empty `DataSources` without an associated `DataImportCron` that SSP creates for each template. Something we can hopefully drop when/if templates go away in the future and instead just rely on CDI and `DataImportCrons` to create and manage `DataSources` for us.